services/web/server/requirements/constraints.txt

```diff
@@ -4,4 +4,8 @@
 #
 
 # SEE services/web/server/tests/unit/isolated/test_utils.py::test_yarl_url_compose_changed_with_latest_release
-yarl<1.6
+yarl>=1.8.2
```

Comment:
Add some doc on why this constraint. Remember the experience we had fixing this issue and try to anticipate what info you might need in order to solve it faster next time

Reply:
`yarl` version `1.8.2` is the first one shipping wheels for Python 3.11

https://yarl.aio-libs.org/en/latest/changes/#id24